packages/webpack/src/config/client.js

```diff
@@ -74,7 +74,7 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
         test: /node_modules[\\/](vue|vue-loader|vue-router|vuex|vue-meta|core-js|@babel\/runtime|axios|webpack|setimmediate|timers-browserify|process|regenerator-runtime|cookie|js-cookie|is-buffer|dotprop|nuxt\.js)[\\/]/,
         chunks: 'all',
         priority: 10,
-        name: true
+        name: 'commons'
```

Comment:
Will this affect  hash name in prod?

Reply:
>  optimization.splitChunks name: true removed: Automatic names are no longer supported

I think was related to this

Reply:
WP5 should give name by default, what’s the name on dev/prod if we remove name: commons

Reply:
![image](https://user-images.githubusercontent.com/5158436/81743868-a7db4780-94a2-11ea-975e-74b42db05f74.png)

@clarkdo seems ok. It is like a hint

Reply:
Great 👍

Reply:
For reference: Removing explicit name, make it hard for bundle analyzer:

![image](https://user-images.githubusercontent.com/5158436/81744946-313f4980-94a4-11ea-991d-26b446718d08.png)